src/main/java/codeu/model/data/User.java

```diff
@@ -41,6 +42,22 @@ public User(UUID id, String name, Instant creation, String hashedPassword, Strin
     this.about = about;
   }
 
+  /**
+   * Constructs a new User to be used during Registration with default about information
```

Comment:
I would add the stuff about "default about information" as a separate line. The spirit of this is that you are creating a bare-bones user for registration, with all the auxilliary info initialized with sane default values (mostly empty strings)

Reply:
Also one more nit. Make sure the docstring is punctuated :)